src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -33203,4 +33207,1228 @@ static class E
             //         return ref E.get_P(ref ri);
             Diagnostic(ErrorCode.ERR_RefReturnNonreturnableLocal, "ri").WithArguments("ri").WithLocation(14, 32));
     }
+
+    [Fact]
+    public void Nullability_Invocation_01()
+    {
+        var src = """
+#nullable enable
+
+object? oNull = null;
+oNull.M();
+
+object? oNotNull = new object();
+oNotNull.M();
+
+static class E
+{
+    extension(object? o)
+    {
+        public void M() { }
+    }
+}
+""";
+        var comp = CreateCompilation(src);
+        comp.VerifyEmitDiagnostics();
+    }
+
+    [Fact]
+    public void Nullability_Invocation_02()
+    {
+        var src = """
+#nullable enable
+
+object? oNull = null;
+oNull.M(); // 1
```

Comment:
>oNull [](http://example.com/codeflow?start=0&length=5)

Do we have tests covering nullability of arguments (both for static and instance extensions)?